draft-ietf-rtcweb-jsep.xml

```diff
@@ -5832,8 +5832,10 @@ a=rtcp-fb:100 nack pli
           "m=" line for that MID.</t>
 
           <t>If the packet's SSRC is in the incoming SSRC mapping
-          table, route the packet to the associated "m=" line and
-          stop.</t>
+          table, check that the packet's PT matches a PT included
+          on the associated "m=" line.  If so, route the packet to
+          that associated "m=" line and stop, otherwise drop the
```

Comment:
If the packet has the SSRC of Transceiver A but the PT does not match, the intent was to drop the packet, not route it to Transceiver B (which does match the PT).  Rereading the text, perhaps this is not clear.  Should we instead have text about the RtpTransceiver examining packets routed to it and dropping packets where the PT does not match?